config/nrf52840/CMakeBuild.config

```diff
@@ -117,27 +117,34 @@ SOFTDEVICE_MINOR=2
 SOFTDEVICE_SERIES=132
 SOFTDEVICE=s132_nrf52_7.2.0
 
-# The microapp based is situated as far to the end of flash as possible.
-# On the nRF52832 this is 19 pages + 4 pages from the end (each page is 0x1000 bytes)
-# Hence, it was 0x80000 - (19+4)*0x1000 = 0x69000.
-# On the nRF52840 we have more pages, but we also want to reserve more for the microapp.
-# Let's assume we want now to reserve not 4, but 16 pages (0x4000, thus 16kB).
-# We can later decide if this is for 4 microapps, or 1 big one.
-#              0x100000 - (19+16)*0x1000 = 0xdd000
-FLASH_MICROAPP_BASE=0xdd000
-
-# Total size of flash is 1024 kB (0x100000 bytes). We can use up to FLASH_MICROAPP_BASE (0xdd000).
-# The maximum size of the app is then 0xdd000 - 0x26000 = 0xb7000
+# The number of pages reserved before FDS for possible FDS expansion in the future
+# On the nRF52832 the microapp size is 2 pages.
+# On the nRF52840 we have more pages, so lets reserve some more.
+CS_FDS_VIRTUAL_PAGES_RESERVED_BEFORE=8
+
+# The microapp located before FDS expansion.
+# On the nRF52832 the microapp size is 4 pages
+# On the nRF52840 we have more pages, so allow for 4 apps.
+# For now though, set number of pages to 4, as the code doesn't work yet for multiple microapps.
+# Page size is 0x1000.
+# 0xEF000 - (4+8+16)*0x1000 = 0xD3000
+FLASH_MICROAPP_BASE=0xD3000
+FLASH_MICROAPP_PAGES=4
+
+# We can use up to the P2P DFU page: FLASH_MICROAPP_BASE - 0x1000.
+# The maximum size of the app is then 0xD9000 - 0x1000 - 0x26000 = 0xB2000
```

Comment:
is this 0xB2000 still correct?

Reply:
Updated, there are too many comments with config values in them :(